cli/src/cli/js-tests-scaffold/index.js

```diff
@@ -3,7 +3,7 @@
 const { Config, Scenario } = require("@holochain/holochain-nodejs")
 Scenario.setTape(require("tape"))
 
-const dnaPath = "./dist/bundle.json"
+const dnaPath = "./dist/YOUR_DNA_NAME.dna.json"
```

Comment:
Is that right?

Reply:
Well, actually we need some automation to edit that file on the fly during `init`.  I'll work on that

Reply:
done